README.md

```diff
@@ -1,210 +1,111 @@
 
-### Examples
+For more detailed options, such as command line switches, see the [Reference Guide](./docs/reference.md)
 
-./examples
+To access sample configurations for commonly-used disk layouts, refer to the [examples](./examples) provided.
 
-### CLI
+## Sample Configuration and CLI command
 
-```
-$ nix run github:nix-community/disko --
-
-disko [options] disk-config.nix
-or disko [options] --flake github:somebody/somewhere
-
-Options:
-
-* -m, --mode mode
-  set the mode, either create or mount
-* -f, --flake uri
-  fetch the disko config relative to this flake's root
-* --arg name value
-  pass value to nix-build. can be used to set disk-names for example
-* --argstr name value
-  pass value to nix-build as string
-* --root-mountpoint /mnt
-  where to mount the device tree
-* --dry-run
-  just show the path to the script instead of running it
-* --debug
-  run with set -x
+A simple disko configuration may look like this:
 
 ```
-
-
-
-## Installing NixOS module
-
-You can use the NixOS module in one of the following ways:
-
-<details>
-  <summary>Flakes (Current recommendation)</summary>
-
-If you use nix flakes support:
-
-``` nix
-{
-  inputs.disko.url = "github:nix-community/disko";
-  inputs.disko.inputs.nixpkgs.follows = "nixpkgs";
-
-  outputs = { self, nixpkgs, disko }: {
-    # change `yourhostname` to your actual hostname
-    nixosConfigurations.yourhostname = nixpkgs.lib.nixosSystem {
-      # change to your system:
-      system = "x86_64-linux";
-      modules = [
-        ./configuration.nix
-        disko.nixosModules.disko
-      ];
-    };
-  };
+{ disks ? [ "/dev/vdb" ], ... }: {  
+ disk = {  
+  vdb = {  
+   device = builtins.elemAt disks 0;  
+   type = "disk";  
+   content = {  
+    type = "table";  
+    format = "gpt";  
+    partitions = [  
+     {  
+      type = "partition";  
+      name = "ESP";  
+      start = "1MiB";  
+      end = "100MiB";  
+      bootable = true;  
+      content = {  
+       type = "filesystem";  
+       format = "vfat";  
+       mountpoint = "/boot";  
+      };  
+     }  
+     {  
+      name = "root";  
+      type = "partition";  
+      start = "100MiB";  
+      end = "100%";  
+      part-type = "primary";  
+      bootable = true;  
+      content = {  
+       type = "filesystem";  
+       format = "ext4";  
+       mountpoint = "/";  
+      };  
+     }  
+    ];  
+   };  
+  };  
+ };  
 }
-```
-</details>
-<details>
-  <summary>niv</summary>
-  
-  First add it to [niv](https://github.com/nmattia/niv):
-
-```console
-$ niv add nix-community/disko
 ```
 
-  Then add the following to your configuration.nix in the `imports` list:
+If you'd saved this configuration in /tmp/disko-config.nix , and wanted to create a disk named /dev/nvme0n1, you would run the following command to partition, format and mount the disk.
```

Comment:
Nitpicks: There's a stray white space before ",", and maybe backticks for `/dev/nvme0n1`?